.github/workflows/ci.yml

```diff
@@ -21,10 +21,8 @@ jobs:
       - name: Check Scala formatting
         if: ${{ always() }}
         run: sbt scalafmtCheckAll
-      - name: Run flaky tests separetly
-        run: sbt 'testOnly *WiderowParquetProcessingSpec'
       - name: Run tests
-        run: sbt coverage 'testOnly * -- -l WiderowParquetProcessingSpec' coverageReport
+        run: sbt coverage test coverageReport
```

Comment:
I assume this means the WiderowParquetProcessingSpec is not so flaky any more?

Reply:
as far as I see, it wasn't flaky before too, I guess it was mistakenly considered flaky previously but @voropaevp can correct me on that

Reply:
It was failing 1/5 runs on github action when run together with the rest of the tests. It could very well be caused by iglu client overload, which is now resolved. But I was not able to isolate the issue at that time.

I think we should merge it back and see how it does.